extensions/aerial-photo/schema.json

```diff
@@ -2,35 +2,57 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://stac.linz.govt.nz/_STAC_VERSION_/aerial-photo/schema.json",
   "title": "Aerial Photography Extension",
-  "description": "STAC Aerial Photography Extension for STAC Items.",
-  "allOf": [
+  "description": "STAC Aerial Photography Extension for STAC Items, and Collection Summaries.",
+  "oneOf": [
     {
-      "type": "object",
-      "required": ["type", "properties", "assets"],
-      "properties": {
-        "type": {
-          "const": "Feature"
-        },
-        "properties": {
-          "allOf": [
-            {
-              "required": ["aerial-photo:run", "aerial-photo:sequence_number"]
+      "$comment": "This is the schema for STAC Items.",
+      "allOf": [
+        {
+          "type": "object",
+          "required": ["type", "properties", "assets"],
+          "properties": {
+            "type": {
+              "const": "Feature"
             },
-            {
-              "$ref": "#/definitions/fields"
+            "properties": {
+              "allOf": [
+                {
+                  "required": ["aerial-photo:run", "aerial-photo:sequence_number"]
+                },
+                {
+                  "$ref": "#/definitions/fields"
+                }
+              ]
+            },
+            "assets": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/fields"
+              }
             }
-          ]
-        },
-        "assets": {
-          "type": "object",
-          "additionalProperties": {
-            "$ref": "#/definitions/fields"
           }
+        },
+        {
+          "$ref": "#/definitions/stac_extensions"
```

Comment:
This is referenced by both items and collections, so I think you can de-dupe a bit by doing something like this (pseudo-JSON):

```
allOf: [
  {"$ref": "#/definitions/stac_extensions"},
  {oneOf: [item, collection]}
]
```

Then you won't need the `allOf` inside both the item and collection definitions, and the STAC extensions reference only appears once. Would that work?

Reply:
I'm not sure if it's better to be efficient or to follow the existing STAC template and examples structure.

Reply:
:shrug: